configs/.env.example

```diff
@@ -1,4 +1,2 @@
-ETH_PRIVATE_KEYS="eeeee6653cdcacc36e3c400ceeeef2aefd59e2642c2f7f298047eeeeeeeeeeee,9643c732204f2a7c9bdb74e2fa08e36d6a4ae8378b983064848b76318fb6507d"
-VIXEOD_KEY=xxx
-CMC_KEY=xxxxxxxxxxxxx
```

Comment:
Where is the comment to explain that can use env vars in the API file?

Reply:
Should I put the comment in the generated doc manually, or somewhere else?